src/fireblocks-sdk.ts

```diff
@@ -89,6 +91,23 @@ export class FireblocksSDK {
         return await this.apiClient.issueGetRequest(url);
     }
 
+    /**
+     * Gets a list of vault accounts per page matching the given filter or path
+     * @param pagedVaultAccountsRequestFilters Filters for the first request
```

Comment:
these 2 params are actually competing \ not being used together.
I'd say it makes more sense to separate this method into 2 methods, each get specific param.
Also, if doing so, the given param must be set or else you should return an error
(e.g.: if the user called such a function and passed no args value its a user error)